critiquebrainz/data/model/user_test.py

```diff
@@ -148,21 +148,21 @@ def test_reviews_property(self):
         self.assertEqual(len(user.reviews), 0)
 
     def test_avatar_property(self):
-        user = User(display_name=u'Tester', email=u"example@example.org")
+        user = User(display_name=u'Tester', email=u"example@example.org", id=u"ea1c2bf3-f246-d47c-f6a4-e5f6e7b8d9e0")
```

Comment:
Hmm, didn't realise that. I did it mostly to make sure the tests are hard-coded, just like they were mostly before.
I'll patch it :)